src/main/java/frc/team2412/robot/commands/intake/IntakeDefaultCommand.java

```diff
@@ -1,17 +1,13 @@
 package frc.team2412.robot.commands.intake;
 
-import static frc.team2412.robot.subsystems.IntakeSubsystem.IntakeConstants.INTAKE_HOLD_SPEED;
-
 import frc.team2412.robot.subsystems.IntakeSubsystem;
-import frc.team2412.robot.subsystems.IntakeSubsystem.IntakeConstants.GamePieceType;
 
 public class IntakeDefaultCommand extends IntakeSetCommand {
 
 	public IntakeDefaultCommand(IntakeSubsystem intakeSubsystem) {
 
-		super(
-				intakeSubsystem,
-				intakeSubsystem.detectType() == GamePieceType.NONE ? 0 : INTAKE_HOLD_SPEED);
+		super(intakeSubsystem, 0);
+		setSpeed(intakeSubsystem.getHoldSpeed());
```

Comment:
I think we will need to set the speed in `initialize()`... my theory is that because the command is only being created once, the speed is only set once, meaning that a game object is added after the constructor is called, the speed won't be updated. The `initialize()` method is called whenever the command is scheduled, which happens constantly since this is the default command and the command finishes instantly.